### DISCUSSION TITLE
How do I remove a section of the card?

### DISCUSSION THREAD

**Discussion by jeremytodd1** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#367] [Q&A] [answered]
I would like to remove this circled section from my lovelace card: https://i.imgur.com/P9MewK2.png

How would I go about doing that? I don't see a way to the documentation to hide this section.

**Comment by PiotrMachowski** [comment, marked as answer]
https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card/discussions/162

**Reply by jeremytodd1** [reply]
Perfect, thank you!
